services/cinema_hall.py

```diff
@@ -0,0 +1,13 @@
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> list[CinemaHall]:
+    return CinemaHall.objects.all()
+
+
+def create_cinema_hall(hall_name: str,
+                       hall_rows: int,
+                       hall_seats_in_row: int) -> CinemaHall:
+    CinemaHall.objects.create(name=hall_name,
+                              rows=hall_rows,
+                              seats_in_row=hall_seats_in_row)

```

Comment:
The `create_cinema_hall` function should return the created `CinemaHall` instance. Consider adding `return CinemaHall.objects.create(...)` to return the instance for further use.